js/modules/loki_snode_api.js

```diff
@@ -129,6 +185,12 @@ class LokiSnodeAPI {
 
     this.onionPaths = [];
     this.guardNodes = [];
+    this.onionRequestCounter = 0; // Request index for debugging
+  }
+
+  getOnionRequestNumber() {
```

Comment:
`getNextOnionRequestNumber` to indicate that we are incrementing? I can see `getOnionRequestNumber` misused by calling it multiple times for the same request.

Reply:
Would changing the name from get to adding help?